File: code/java/Leapyear.java
import java.util.*;
class Leapyear{
  public static void main(String[] args) {
    Scanner sc= new Scanner(System.in);
    int year;
    System.out.println("Enter the year");
    year = sc.nextInt();
    if(year % 4 == 0){
      System.out.println("Entered year is Leap Year");
    }
    else{
      System.out.println("Entered year is Not a Leap Year");
    }
  }
}

File: code/c/leapyear.c
#include <stdio.h>

void main() 
{
  int year;
  printf("Enter the year\n");
  scanf("%d",&year);
  if(year % 4 == 0)
  {
    printf("Entered year is Leap Year");
  }
  else
  {
    printf("Entered year is Not a Leap Year");
  }
}